config/config.go

```diff
@@ -67,7 +65,7 @@ func init() {
 // Load will load the necessary config
 func Load(logger logrus.FieldLogger) (Options, error) {
 	if err := viper.ReadInConfig(); err != nil {
-		logger.Warn("unable to find config.yml. loading config from environment variables")
+		logger.Warn(err)
```

Comment:
better to `errors.Wrap(err, "<the old mesasge>")` so it's clear what's the problem. Error of viper won't be really helpful